hookman/hookman_generator.py

```diff
@@ -711,6 +712,31 @@ def _generate_windows_body(hooks):
         f"    }}",
         f"",
         f"",
+        f"    HMODULE Load_DLL(const std::wstring& w_filename) {{",
+        f'        wchar_t *path_env = NULL;',
+        f'        rsize_t path_env_len = 0;',
+        f'        // Read PATH environment variable',
+        f'        _wdupenv_s(&path_env, &path_env_len, L"PATH");',
+        f'        std::wstring::size_type dir_name_size = w_filename.find_last_of(L"/\\\\");',
+        f'        int size = path_env_len + MAX_PATH;',
+        f'        wchar_t *new_path_env;',
+        f'        new_path_env = (wchar_t *)malloc(size);',
```

Comment:
@tadeu better use `new` here? Also, this is not being freed.

Reply:
I removed this code.

Reply:
Sorry, I was in meeting all day and I'm late to the party here, it seems everything is good now, with `wstring`s :)